appengine-jetty-managed-runtime/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -238,7 +254,7 @@ public void init(String appengineWebXmlFile)
       AppEngineWebXmlReader appEngineWebXmlReader
               = new AppEngineWebXmlReader(appDir, appengineWebXmlFile);
       appEngineWebXml = appEngineWebXmlReader.readAppEngineWebXml();
-   }
+    }
     VmRuntimeUtils.installSystemProperties(defaultEnvironment, appEngineWebXml);
     VmRuntimeLogHandler.init();
     VmRuntimeFileLogHandler.init();
```

Comment:
@ludoch this is where the JVM logging is configured and it will use the JSON formatter in the file log handler.  See other note for jetty logging configuration.